scenes/player_test/states/left.gd

```diff
@@ -0,0 +1,8 @@
+extends CommonFeaturesState
+
+
+func on_input(event):
+	# seria mejor usar el parametro _event para obtener la información del evento
+	if Input.is_action_just_pressed("ui_left"): 
+		state_machine.change_to("Right")
+		print("right")
```

Comment:
I think we won't use these actions in the final version

Reply:
yep there is examples, i think we need other branch to upload these kilnd of example code.